etc/schema.yaml

```diff
@@ -82,16 +82,42 @@ samples:
     properly_mapped_reads: {    type: 'integer', required: False }
     duplicate_reads: {          type: 'integer', required: False }
     median_coverage: {          type: 'float',   required: False }
-    coverage: {                 type: 'dict',    required: False, schema: { median: {type: 'float'}, mean: {type: 'float'}, std_dev: {type: 'float'} } }
+    coverage: {                 type: 'dict',    required: False, schema: {
+        median: {type: 'float'},
+        mean: {type: 'float'},
+        std_dev: {type: 'float'},
+        percentiles: {type: 'dict'},
+        pc_genome_with_min: {type: 'dict'}
```

Comment:
There doesn't seem to be anything in [Analysis Driver's pull request](https://github.com/EdinburghGenomics/Analysis-Driver/pull/178) for these new fields. We should also give them some schema structure if possible.